tests/Argu.Tests/Tests.fs

```diff
@@ -474,6 +483,12 @@ module ``Argu Tests Main List`` =
         raisesWith<ArguParseException> <@ parser.ParseCommandLine args @>
                                     (fun e -> <@ e.FirstLine.Contains "--branch" @>)
 
+    [<Fact>]
+    let ``Main command parsing should fail and display all missing mandatories sub command parameters`` () =
+        let args = [|"--mandatory-arg" ; "true" ;  "multiple-mandatories" ; "--valuea"; "5"|]
+        raisesWith<ArguParseException> <@ parser.ParseCommandLine(args, raiseOnUsage = true) @>
```

Comment:
```suggestion
        raisesWith<ArguParseException> <@ parser.ParseCommandLine args @>
```

Reply:
done

Reply:
is it :P

Reply:
actually, I only changed it on another test and didn't pay attention to the other one 😅
done

Reply:
And I completely forgot about this PR!
Assuming you don't shout stop, I'll merge shortly